server/src/main/java/org/eclipse/lsp/cobol/core/engine/dialects/idms/ErrorHelper.java

```diff
@@ -44,4 +44,19 @@ public SyntaxError reportMissingCopybooks(MessageService messageService, Localit
     LOG.debug("Syntax error by reportMissingCopybooks: {}", error.toString());
     return error;
   }
+
+  public SyntaxError reportCircularDependency(MessageService messageService, Locality locality, String copybookName) {
```

Comment:
How do you feel about dropping "report" from method names?

Reply:
I am okay with that!